src/test/java/teammates/test/cases/browsertests/AdminHomePageUiTest.java

```diff
@@ -101,6 +101,34 @@ private void testCreateInstructorAction() throws Exception {
                      homePage.getMessageFromResultTable(2));
         homePage.clearInstructorDetailsSingleLineForm();
         
+        ______TS("action success: displayed instructor details are properly HTML-encoded");
+        
+        InstructorAttributes dangerousInstructor = new InstructorAttributes();
+        
+        String shortNameDangerous = "<b>MaliciousInstrúctör</b>";
+        dangerousInstructor.name = "Malicious <script>alert('dangerous');</script>Instrúctör";
+        dangerousInstructor.email = "malicious.instr1<>!@gmail.tmt";
+        String dangerousInstitute = "TEAMMATES Malicious Institute <!@!@!>";
+        String dangerousDemoCourseId = "malicious.instr1___.gma-demo";
+        
+        BackDoor.deleteAccount(TestProperties.TEST_INSTRUCTOR_ACCOUNT);
+        BackDoor.deleteCourse(dangerousDemoCourseId);
+        BackDoor.deleteInstructor(dangerousDemoCourseId, dangerousInstructor.email);
+        
+        homePage.createInstructor(shortNameDangerous, dangerousInstructor, dangerousInstitute);
+        
+        encryptedKey = BackDoor.getEncryptedKeyForInstructor(dangerousDemoCourseId, dangerousInstructor.email);
```

Comment:
I don't think this block is needed as we're not checking the expected join URL in this test